Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py

```diff
@@ -638,7 +638,8 @@ def parse_manifest_and_object_args() -> tuple:
             datetime.strptime(added_after, UTC_DATE_FORMAT)
     except ValueError:
         try:
-            datetime.strptime(added_after, STIX_DATE_FORMAT)
+            if added_after:
```

Comment:
There won't be a ValueError unless the `if added after` in L637 evaluates to `True`. Is L641 mypy's requirement?

Reply:
```
Argument 1 to "strptime" of "datetime" has incompatible type "str | None";
expected "str"  [arg-type]
                datetime.strptime(added_after, STIX_DATE_FORMAT)
```